projects/plugins/jetpack/extensions/blocks/conversation/components/participants-controls.js

```diff
@@ -24,22 +24,22 @@ function AddParticipantButton( { className, onAdd, participants = [] } ) {
 function ParticipantsLabelControl( { className, participants, onChange, onDelete } ) {
 	return (
 		<div className={ `${ className }__participant-control` }>
-			{ participants.map( ( { participant, participantSlug } ) => (
-				<div key={ `${ participantSlug }-key` } className={ `${ className }__participant` }>
+			{ participants.map( ( { label, slug } ) => (
+				<div key={ `${ slug }-key` } className={ `${ className }__participant` }>
 					<TextControl
-						value={ participant }
-						onChange={ participantEditedValue =>
+						value={ label }
+						onChange={ value =>
 							onChange( {
-								participantSlug,
-								participant: participantEditedValue,
+								slug,
+								label: value,
 							} )
 						}
 					/>
 
 					<Button
 						className={ `${ className }__remove-participant` }
 						label={ __( 'Remove participant', 'jetpack' ) }
-						onClick={ () => onDelete( participantSlug ) }
+						onClick={ () => onDelete( slug ) }
```

Comment:
Updates participant selection controls to use new participant object shape with `label` and `slug`.